setup.py

```diff
@@ -6,6 +6,7 @@
 
 
 REQUIREMENTS = [
+    'Django<3.1',
```

Comment:
please don't add a django restriction here

Reply:
maybe we could just wait for #543 to pass and then rebase this

Reply:
This was a test, because Travis-CI failed for reasons I can not retrace.

Reply:
👍

Reply:
:+1: